examples/cfd/08_shallow_water_equation.ipynb

```diff
@@ -0,0 +1,4210 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    print(optime.ccode)
</pre><p>optime can be renamed as op</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/1867/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>